lib/services/common/securityServiceOAuth2.js

```diff
@@ -21,6 +21,8 @@
  * For those usages not covered by the GNU Affero General Public License
  * please contact with::daniel.moranjimenez@telefonica.com
  * This code was contributed by Federico M. Facca on behalf of Martel Innovate
```

Comment:
Should this PR involves changes in configuration or documentation?

Reply:
Fixed with f55e7fb337fb4f14eb26f8c021aca3fe503cd44d